src/components/ContactList/ContactList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import './ContactList.css';
+import Contact from '../Contact';
+
+function ContactList() {
+  const contacts = [
+    { id: 1, link: '+375293837184', text: 'Contact us by Phone', to: 'tel:' },
+    {
+      id: 2,
+      link: '375293837184',
+      text: 'Contact us by WhatsApp',
+      to: 'https://api.whatsapp.com/send?phone=',
+    },
+    { id: 3, link: 'peterbabukh', text: 'Contact us by Skype', to: 'skype:' },
+    {
+      id: 4,
+      link: 'babukhpeter3@gmail.com',
+      text: 'Write us an E-mail',
+      to: 'mailto:',
+    },
+  ];
+
+  return (
+    <ul className="contact-list">
```

Comment:
Again, this component is not needed. A class method of sidebar is enough. I do not think you will use it again this way somewhere else.